sdk/python/feast/telemetry.py

```diff
@@ -30,29 +32,38 @@
 
 class Telemetry:
     def __init__(self):
-        feast_home_dir = join(expanduser("~"), ".feast")
-        Path(feast_home_dir).mkdir(exist_ok=True)
-        telemetry_filepath = join(feast_home_dir, "telemetry")
-        self._telemetry_enabled = (
+        self._telemetry_enabled = False
+        self.check_env()
+
+    def check_env(self):
```

Comment:
`check` is not an appropriate name if we are actually writing and initializing Telemetry.